support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -201,13 +201,15 @@ export function ThreeTierCard({
 	const promoPriceCopy = `${currency}${promoPrice}/${recurringContributionPeriodMap[paymentFrequency]}`;
 	const quantumMetricButtonRef = `tier-${cardTier}-button`;
 	return (
-		<div css={container(isRecommended, isUserSelected, isRecommendedSubdued)}>
+		<section
```

Comment:
Presume as we grouped together everything inside to a TierCard.

Reply:
I was thinking these might actually be better as a list with a heading "Products".

I have used a process previously which is to strip all CSS and see if your page makes sense. Fancy pairing on that?